salt/modules/debbuild.py

```diff
@@ -23,6 +23,8 @@
 from salt.exceptions import SaltInvocationError
 from salt.ext.six.moves.urllib.parse import urlparse as _urlparse  # pylint: disable=no-name-in-module,import-error
 
+import salt.modules.gpg
```

Comment:
This is not the correct way to do this. If the necessary libs, (i.e., `gnupg`) are not present on the system, function in this module will not operate properly. Instead, various functions needs to be cross-called via `__salt__['gpg.my_func']()`